packages/style-engine/src/test/index.js

```diff
@@ -255,6 +267,26 @@ describe( 'getCSSRules', () => {
 				key: 'backgroundColor',
 				value: '#555555',
 			},
+			{
```

Comment:
LGTM  👍
How about unifying the properties of this matcher object in the order `selector` / `key` / `value`?

Reply:
Haha, I can also copy paste 😄 

Good catch, will do.